tests/integrationtest/t/planner/core/casetest/partition/integration_partition.test

```diff
@@ -131,6 +131,7 @@ insert into tto_seconds values ('2019-12-31 12:00:00', '2019-12-31 23:59:59'), (
 select * from tto_seconds;
 --sorted_result
 select * from tto_seconds partition(p0);
+set @@tidb_partition_prune_mode = 'dynamic';
 
 # TestEstimationForTopNPushToDynamicPartition
```

Comment:
You can see the next test called ```TestEstimationForTopNPushToDynamicPartition```. but at the front, it has enable stats prune mode. so i enable dynamic partition.